elements/pfe-band/src/pfe-band.scss

```diff
@@ -68,9 +55,8 @@ $LOCAL: band;
 
   // Wrapper sets padding and background visuals
   position: relative;
-  @extend %flex-fallback;
 
-  // Padding 
+  // Padding
```

Comment:
Should I open a new PR with the vars that make this match the design system? or is that fair game for this PR?

![image](https://user-images.githubusercontent.com/456863/68235808-e531ba80-ffd1-11e9-9b52-da17491e0f8a.png)

![image](https://user-images.githubusercontent.com/456863/68235836-ee228c00-ffd1-11e9-8e89-27909de15d7b.png)

Reply:
I think new PR - this has already been tested so I want to be careful about expanding scope.  This is ready for merge as-is.

Reply:
Also for variable naming: `--pfe-band__body--gutter--vertical` < regions are indicated by underscores

Reply:
moved here https://github.com/patternfly/patternfly-elements/pull/609